command/certificate/install.go

```diff
@@ -0,0 +1,205 @@
+package certificate
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/command"
+	"github.com/smallstep/cli/crypto/pemutil"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/ui"
+	"github.com/smallstep/truststore"
+	"github.com/urfave/cli"
+)
+
+func installCommand() cli.Command {
+	return cli.Command{
+		Name:   "install",
+		Action: command.ActionFunc(installAction),
+		Usage:  "install a certificate in the system truststore",
+		UsageText: `**step certificate install** <crt-file>
+		[**--prefix**=<name>] [**--all**]
+		[**--java**] [**--firefox**] [**--no-system**]`,
+		Description: `**step certificate install** installs a certificate in the system truststore.
```

Comment:
installs a *root* certificate. we should be explicit here (for dumb people like me).